proxy/adminservice.go

```diff
@@ -213,21 +225,56 @@ func (s *adminServiceProxyServer) StreamWorkflowReplicationMessages(
 	if !ok {
 		return serviceerror.NewInvalidArgument("missing cluster & shard ID metadata")
 	}
-	targetClusterShardID, sourceClusterShardID, err := history.DecodeClusterShardMD(
```

Comment:
Not sure if this was my fault when I bumped the temporal version, but the target/source ids here were flipped.

I changed them to match "client/server" terminology of the [`history.DecodeClusterShardMD`](https://github.com/temporalio/temporal/blob/e04b9d79a2fbd09422a64f9fcc3f73410dbe89cc/client/history/metadata.go#L62-L64) function?

I'm fine either way.